src/webgl/p5.RendererGL.js

```diff
@@ -468,6 +473,17 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
     this.spotLightAngle = [];
     this.spotLightConc = [];
 
+    // Null if no image light is set, otherwise, it is set to a p5.Image
```

Comment:
Maybe we can rephrase this into sentences so it's a bit easier to read? Also, for each map, we should say what the keys and values are. E.g. for this one, it maps a `p5.Image` used by `imageLight()` to a `p5.Graphics` containing the combined light it sends to each direction.